cli/commands/pods.go

```diff
@@ -56,55 +70,157 @@ func toPodTree(podsJSONBytes []byte) string {
 	return buf.String()
 }
 
-func appendPod(buf *bytes.Buffer, podName string, rawPod interface{}, lastPod bool) {
-	var podPrefix string
+func appendServicePod(buf *bytes.Buffer, rawPod interface{}, lastPod bool) {
+	pod, ok := rawPod.(map[string]interface{})
+	if !ok {
+		return
+	}
+
+	var prefix string
+	if lastPod {
+		prefix = "└─ "
+	} else {
+		prefix = "├─ "
+	}
+
+	buf.WriteString(fmt.Sprintf("%s%s\n", prefix, pod["name"]))
+
+	rawPodInstances, ok := pod["instances"].([]interface{})
+	if !ok {
+		return
+	}
+	for i, rawPodInstance := range rawPodInstances {
+		appendServicePodInstance(buf, rawPodInstance, lastPod, i == len(rawPodInstances)-1)
+	}
+}
+
+func appendServicePodInstance(buf *bytes.Buffer, rawPodInstance interface{}, lastPod bool, lastPodInstance bool) {
```

Comment:
Probably out of scope of this PR, but there has to be a neater way of doing this nesting. It would be good to be able to use the same function here. Especially considering how nested the "if last" structures get.

Reply:
Cleaned this up somewhat: the parent is now passing down a prefix to be used in the output.

Reply:
Already looking a lot better. I found: 
https://github.com/campoy/tools/blob/master/tree/main.go
and
https://github.com/a8m/tree/blob/master/node.go
which may allow for a more general implementation. As I say -- really outside of the scope of this PR. Thanks for the effort.